app/forms/frontend_error_form.rb

```diff
@@ -0,0 +1,30 @@
+class FrontendErrorForm
+  include ActiveModel::Model
+  include ActionView::Helpers::TranslationHelper
+
+  validate :validate_filename_extension
+  validate :validate_filename_host
+
+  attr_reader :filename
+
+  def submit(filename:)
+    @filename = filename
+
+    FormResponse.new(success: valid?, errors:, serialize_error_details_only: true)
+  end
+
+  private
+
+  def validate_filename_extension
+    return if File.extname(filename.to_s) == '.js'
```

Comment:
do we 100% always get the filename? my guess is we'd want to allow a `nil` filename too just in case?

Reply:
The [spec](https://html.spec.whatwg.org/multipage/webappapis.html#the-errorevent-interface) implies that it must be a string, but that it _could_ (at least by the defaults defined there) be empty. The behavior here matches [the behavior on the client-side](https://github.com/18F/identity-idp/blob/c099dc78a30532e7d473156976fabbc15c8907ac/app/javascript/packages/analytics/is-trackable-error-event.ts#L3), which would ignore anything not explicitly `.js`. Whether we should want to log those, I'd have a mild curiosity what (if anything) would be logged, but generally think we're interested in errors known to be originating from our own scripts.